tests/test_bfh.py

```diff
@@ -172,9 +172,9 @@ def test_can_init_arrays_with_dict(self):
 
     def test_implicit_nulls(self):
         """Implicit nulls by default"""
-        result = Conversation().serialize()
+        result = Conversation().serialize(implicit_nulls=True)
         self.assertEqual({}, result)
-        result = Conversation(numbers=[1, 2]).serialize()
+        result = Conversation(numbers=[1, 2]).serialize(implicit_nulls=True)
         self.assertEqual({"numbers": [1, 2]}, result)
         result = Conversation().serialize(implicit_nulls=False)
```

Comment:
can you add a separate test that illustrates that the default behavior is now False?